h2o-py/h2o/h2o.py

```diff
@@ -248,22 +248,25 @@ def get_mem_size(mmint, mmgb):
     h2oconn.cluster.show_status()
 
 
-def lazy_import(path):
+def lazy_import(path, pattern):
     """
     Import a single file or collection of files.
 
     :param path: A path to a data file (remote or local).
+    :param pattern: Character string containing a regular expression to match file(s) in the folder.
```

Comment:
Not clear how the two parameters interact with each other. The `path` parameter is documented to be the "path to a data file". At the same time `pattern` is a "regular expression" describing a set of files. So -- can they be used simultaneously? If not, then there ought to be a check for that. If yes, then the meaning should be clarified (in particular, when `path` is an array of filenames)